bats/tests/helpers/vm.bash

```diff
@@ -150,30 +150,65 @@ start_application() {
 
 container_engine_info() {
     run ctrctl info
-    assert_success
+    echo "$output"
```

Comment:
Why the `echo`, and shouldn't it be `echo ... 1>&3` if you really want to output.  Same with another new `echo $output` in vm.bash and an existing one in `utils.bash`.

Reply:
It make sure that the `$output` is propagated to the caller when you run it via `try`:

```shell
try container_engine_info
assert_output ...
```

Because `try` uses `run` internally, so does not preserve `$output` from the command it runs.

It doesn't really matter right now because we don't use the output in the current code, but at least it will work when/if we ever want to.

The function name maybe should be changed to `get_container_engine_info`, but I will only do that if there is other actionable feedback on the PR.